src/api/productAPI.ts

```diff
@@ -118,3 +118,39 @@ export async function getKeydeukBest() {
     throw error;
   }
 }
+
+export const getRecentProducts = async (): Promise<RecentProductType[]> => {
+  const token = await getCookie('accessToken');
+
+  try {
+    const res = await fetch(`${BASE_URL}/api/v1/user/recent-products`, {
+      cache: 'no-cache',
+      headers: {
+        'Content-Type': 'application/json',
+        Authorization: token ? `Bearer ${token}` : '',
+      },
+    });
+
+    const result = await res.json();
+
+    return result.data;
+  } catch {
+    throw new Error(`최근 본 상품을 조회할 수 없습니다. `);
+  }
```

Comment:
```suggestion
  } catch (error){
    throw error;
  }
```
전에 디버깅을 하는게 아니라면 콘솔창에서 이런식의 피드백이 나오는게 좋은거같지 않다고 이야기 했던게 기억납니다..!